src/extensionApi.ts

```diff
@@ -176,12 +176,58 @@ export interface TeamOverrides {
   relationships: Relationship[];
 }
 
+export interface Rule {
+  id: number;
+  expression: string;
+  title?: string;
+  description?: string;
+  failureMessage?: string;
+  dateCreated?: string;
+  weight: number;
+}
+
+export interface ServiceGroup {
```

Comment:
should we name this group: 

```suggestion
export interface Group {
```

Reply:
ahh yes!